tools/lsp/ui/components/widgets/brush-widget.slint

```diff
@@ -104,47 +107,50 @@ export component BrushWidget inherits GridLayout {
         }
     }
 
-    Row {
-        kind_cb := ComboBox {
-            col: 1;
+    // Row {
+    //     kind_cb := ComboBox {
+    //         col: 1;
 
-            model: [ "Solid Color", "Linear Gradient", "Radial Gradient"];
+    //         model: [ "Solid Color", "Linear Gradient", "Radial Gradient"];
 
-            changed current-index => {
-                if self.current-index == 0 {
-                    root.current-brush-kind = BrushKind.solid;
-                } else if self.current-index == 1 {
-                    root.current-brush-kind = BrushKind.linear;
-                } else {
-                    root.current-brush-kind = BrushKind.radial;
-                }
+    //         changed current-index => {
+    //             if self.current-index == 0 {
+    //                 root.current-brush-kind = BrushKind.solid;
+    //             } else if self.current-index == 1 {
+    //                 root.current-brush-kind = BrushKind.linear;
+    //             } else {
+    //                 root.current-brush-kind = BrushKind.radial;
+    //             }
 
-                root.update-brush();
-            }
-        }
-    }
+    //             root.update-brush();
+    //         }
+    //     }
+    // }
 
```

Comment:
We have version control, so you do not need to keep commented out code around :-)